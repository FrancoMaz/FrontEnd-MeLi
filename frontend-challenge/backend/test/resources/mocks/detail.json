{
  "id": "ML1",
  "title": "Libro 1",
  "price": 3500,
  "pictures": [
    {
      "secure_url": "http://image.jpg"
    }
  ],
  "currency_id": "ARS",
  "shipping": {
    "free_shipping": true
  },
  "condition": "new",
  "sold_quantity": 20,
  "category_id": "MLA412445"
}