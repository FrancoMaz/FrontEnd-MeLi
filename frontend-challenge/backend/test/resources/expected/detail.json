{
  "author": {
    "name": "Franco",
    "lastname": "Mazzoni"
  },
  "item": {
    "id": "ML1",
    "title": "Libro 1",
    "price": {
      "currency": "$",
      "amount": 3500,
      "decimals": 2
    },
    "picture": "http://image.jpg",
    "condition": "new",
    "freeShipping": true,
    "soldQuantity": 20,
    "stateName": null,
    "categories": [
      "Libros, Revistas y Comics",
      "Libros"
    ],
    "description": "Este es un libro"
  }
}